community/nextcloud-client/APKBUILD

```diff
@@ -1,31 +1,31 @@
 # Maintainer: Leonardo Arena <rnalrd@alpinelinux.org>
 pkgname=nextcloud-client
 pkgver=2.5.2
-pkgrel=0
+pkgrel=1
 pkgdesc="Nextcloud Desktop Client"
-url="https://github.com/nextcloud/client_theming"
-arch="all !x86 !s390x !ppc64le" # qt5-qtwebengine not avail
+url="https://github.com/nextcloud/desktop"
+arch="all"
 license="GPL-2.0-only"
-makedepends="$depends_dev cmake doxygen qt5-qtwebengine-dev qt5-qttools-dev
+makedepends="$depends_dev cmake doxygen qt5-qttools-dev
 	qt5-qtwebkit-dev qt5-qtkeychain-dev"
 subpackages="$pkgname-dev"
 options="!check" # no testsuite
-source="nextcloud-client.tar.gz::https://github.com/nextcloud/desktop/archive/v$pkgver.tar.gz"
+source="nextcloud-client.tar.gz::https://github.com/nextcloud/desktop/archive/v$pkgver.tar.gz
+	remove-qtwebengine.patch"
 builddir="$srcdir"/desktop-$pkgver
 
 build() {
-	cd "$builddir"
 	cmake \
-		-DCMAKE_BUILD_TYPE="Release" \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
 		-DCMAKE_INSTALL_LIBDIR=lib \
 		-DCMAKE_INSTALL_PREFIX=/usr \
 		-DCMAKE_INSTALL_SYSCONFDIR=/etc/$pkgname
```

Comment:
I guess we should add `-DNO_SHIBBOLETH=1` here to get rid of the qt5-qtwebkit dependency, upstream has supported app passwords for some years now (and qt5-qtwebkit is super insecure at this point).